### PR TITLE
Proposed implementation of #106

### PR DESCRIPTION
**Pull request by maximelefrancois86**
As part of our work at ETSI on the development of a new version of SAREF, we reached last week an agreement on how to solve this issue.

Note that this is a compromise that resolves a six year long dispute between two former contributors to SOSA/SSN 😉, so I hope it will be well received by the group.

* https://raw.githack.com/w3c/sdw-sosa-ssn/property-of-interest/ssn/index.html#SSNProperty
* https://raw.githack.com/w3c/sdw-sosa-ssn/property-of-interest/ssn/index.html#SSNPropertyOfInterest

### CODE REVIEW

File: ssn/integrated/sosa.ttl [hunk 1]
```diff
@@ -70,10 +70,20 @@ sosa:FeatureOfInterest a rdfs:Class , owl:Class ;
...
+  skos:note "Concepts from existing code lists, vocabularies, and taxonomies, may be used as instances of Property"@en ;
   rdfs:isDefinedBy sosa: .
 
+sosa:PropertyOfInterest a owl:Class ;
+  rdfs:label "Property"@en ;
+  skos:definition "An identifiable quality of a FeatureOfInterest that can be observed or acted upon. A property of interest is specific to a feature of interest. It is inherent to and cannot exist without that FeatureOfInterest."@en ;
+  rdfs:comment "An identifiable quality of a FeatureOfInterest that can be observed or acted upon. A property of interest is specific to a feature of interest. It is inherent to and cannot exist without that FeatureOfInterest."@en ;
+  skos:example "Abby's car has the colour red, where \"colour red\" is a property of interest of the car."@en ;
+  skos:note "Properties of interest need not always be explicitly identified. It depends on the use case."@en ;
+  rdfs:isDefinedBy ssn: .
```
Comment (dr-shorthair) [inline ssn/integrated/sosa.ttl]: sosa: ?

File: ssn/integrated/ssn.ttl [hunk 2]
```diff
@@ -59,9 +59,21 @@ sosa:FeatureOfInterest
 ssn:Property a owl:Class ;
```
Comment (dr-shorthair) [inline ssn/integrated/ssn.ttl]: This class is now deprecated. Should the axiomatization still be updated? 

Maybe better to add the axioms to `sosa:Property` instead. 

Is there also an axiomatization of `sosa:PropertyOfInterest` that should be added to this graph?

Comment (maximelefrancois86) [reply ssn/integrated/ssn.ttl]: Clarified in new commit 351438d

File: ssn/integrated/sosa.ttl (hunk 1 above)
Comment (maximelefrancois86) [reply ssn/integrated/sosa.ttl]: corrected in commit 351438d

**Review by dr-shorthair — APPROVED** [review]

**Comment by dr-shorthair** [comment]
Also see #49 and #52

Closes #106

**Review by rob-metalinkage — CHANGES_REQUESTED** [review]
Label of PropertyOfInterest is just "Property"

is there a need for a PropertyKind as well for symmetry to allow for more general classifications? And potentially a constraint for ObservationCollections that observedProperty objects are some form of specialisation of PropertyKind?  Is there an equivalent constraint for Property w.r.t to PropertyOfInterest ? 

also we have identified 3 possible profiles for observedProperty depending on which case - is the range of observedProperty a union of these options or do we need more predicates?  

examples are critical here me feels..

**Comment by maximelefrancois86** [comment]
@rob-metalinkage I fixed the nit in latest commit on this branch.

If in addition to "PropertyOfInterest" you believe we would need a third class named "PropertyKind", I suggest you approve this PR first, then we could discuss that need.

I intentionally left out any modification of the ranges of object properties, to proceed incrementally

**Review by rob-metalinkage — APPROVED** [review]

**Pull request merged**